sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/ExpressionConverter.java

```diff
@@ -644,7 +649,7 @@ private RexNode convertResolvedFunctionCall(
           throw new UnsupportedOperationException(
               "Unsupported function: " + funName + ". Only support TUMBLE, HOP, and SESSION now.");
       }
-    } else if (funGroup.equals("ZetaSQL")) {
+    } else if (funGroup.equals(ZETASQL_FUNCTION_GROUP_NAME)) {
```

Comment:
nit: ZETASQL_FUNCTION_GROUP_NAME.equals(funGroup) (There are a bunch more of this pattern, I'm going to stop commenting on them.)

Reply:
Thanks, forgot about that. I fixed this part in #13309.